docs/security.html

```diff
@@ -547,12 +547,20 @@ <h3><a id="security_sasl" href="#security_sasl">7.3 Authentication using SASL</a
             <ul>
             <li>SASL/PLAIN should be used only with SSL as transport layer to ensure that clear passwords are not transmitted on the wire without encryption.</li>
             <li>The default implementation of SASL/PLAIN in Kafka specifies usernames and passwords in the JAAS configuration file as shown
-                <a href="#security_sasl_plain_brokerconfig">here</a>. To avoid storing passwords on disk, you can plug in your own implementation of
-                <code>javax.security.auth.spi.LoginModule</code> that provides usernames and passwords from an external source. The login module implementation should
+                <a href="#security_sasl_plain_brokerconfig">here</a>. From Kafka version 2.0 onwards, you can avoid storing clear passwords on disk
+                by configuring your own callback handlers that obtain username and password from an external source using the configuration options
+                <code>sasl.server.callback.handler.class</code> and <code>sasl.client.callback.handler.class</code>.
+                <p>
+                With older versions of Kafka, you can plug in your own implementation of <code>javax.security.auth.spi.LoginModule</code>
+                that provides usernames and passwords from an external source. The login module implementation should
                 provide username as the public credential and password as the private credential of the <code>Subject</code>. The default implementation
-                <code>org.apache.kafka.common.security.plain.PlainLoginModule</code> can be used as an example.</li>
```

Comment:
We normally just document the newest version since we keep the older version docs.

Reply:
@ijuma Thanks for the review, removed section for older versions.